src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -514,8 +506,6 @@ public DownloadInstance tabularDatafileMetadataPreprocessed(@PathParam("fileId")
             logger.fine("Preprocessed data for tabular file "+fileId);
         }
         
-        response.setHeader("Access-Control-Allow-Origin", "*");
-        
```

Comment:
This "dataaccess" API is subject to ":AllowCors", right? I assume so since it looks like the setting affects a low level, the ApiBlockingFilter.

Reply:
I think so. I did not test that specific API but  I tested tabularDatafileMetadata API from dataaccess and it works with the new setting.

Reply:
Ok, it should be easy enough to test, right? With `curl -i` you can see the headers.

Reply:
Ok, thanks, I just run it and it has the required headers.